package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "de.appplant.cordova.plugin.local-notification",
   "cordova_name": "Cordova LocalNotification Plugin",
-  "version": "0.8.9.OS",
+  "version": "0.8.9-OS3",
```

Comment:
The idea going forward is to keep master as a mirror of upstream, hence this version should only be present in the `outsystems` branch.

Reply:
I know that, but unfortunately the damage was already done in this repository. I just bumped the version in the master branch because it already has OS changes. The version had already been changed as well.

Reply:
Indeed it had been changed already. Most plugins with customisations in `master` branch hail from a time before it was settled that such custom changes would happen only in an `ousystems` branch.

In fact it is not out of the question a complete rewrite of the `master` branch to make it mirror `upstream:master` 100%.

Therefore whether or not the suffix `OS` is present in the `master` branch is moot. Only wanted to save you some time in your workflow.

It is very likely that I overlooked this situation in other repositories, so feel free to spread the word as typing this on every single one of them would not be practical.